src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2534,7 +2546,104 @@ namespace winrt::TerminalApp::implementation
     {
         for (uint32_t i = 0; i < _tabs.Size(); ++i)
         {
-            _GetStrongTabImpl(i)->UpdateTabViewIndex(i);
+            _tabs.GetAt(i).UpdateTabViewIndex(i);
+        }
+    }
+
+    void TerminalPage::_OpenSettingsUI()
```

Comment:
doc comment

Reply:
Specifically call out in the method comment that if the Settings UI is already open, we just switch to it.